test/unit/app/services/bulk_republisher_test.rb

```diff
@@ -201,4 +201,40 @@ class BulkRepublisherTest < ActiveSupport::TestCase
       BulkRepublisher.new.republish_all_published_organisation_about_us_pages
     end
   end
+
+  describe "#republish_all_by_type" do
+    setup do
+      BulkRepublisher.any_instance.stubs(:non_editionable_content_types).returns(%w[Contact])
+    end
+
+    context "for editionable content types, like CaseStudy" do
+      test "republishes all content of the specified type via the PublishingApiDocumentRepublishingWorker" do
+        content_type = create(:case_study)
```

Comment:
Again this is an instance or record, so I find `content_type` a little misleading as a variable name (same below)